features/bevragen/persoon-beperkt/verblijfplaats/dev/fields-gba.feature

```diff
@@ -0,0 +1,394 @@
+    | 080910                  | verblijfplaats                                         | gemeente van inschrijving     |
+    | 081410                  | verblijfplaats.functieAdres                            | land vanwaar ingeschreven     |
+
+  Abstract Scenario: '<type>' van een locatie is in onderzoek en één of meerdere velden wordt gevraagd met field pad '<fields>'
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende gegevens
+    | naam                  | waarde   |
+    | geslachtsnaam (02.40) | Maassen  |
+    | geboortedatum (03.10) | 19830526 |
+    En de persoon heeft de volgende 'verblijfplaats' gegevens
+    | naam                            | waarde                    |
+    | aanduiding in onderzoek (83.10) | <aanduiding in onderzoek> |
+    | datum ingang onderzoek (83.20)  | 20020701                  |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | naam                        | waarde                      |
+    | gemeentecode (92.10)        | 0518                        |
+    | locatiebeschrijving (12.10) | Woonboot bij de Grote Sloot |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam          | waarde                              |
+    | type          | ZoekMetGeslachtsnaamEnGeboortedatum |
+    | geslachtsnaam | Maassen                             |
+    | geboortedatum | 1983-05-26                          |
+    | fields        | <fields>                            |
+    Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+    | naam                                      | waarde                      |
+    | locatiebeschrijving                       | Woonboot bij de Grote Sloot |
+    | inOnderzoek.aanduidingGegevensInOnderzoek | <aanduiding in onderzoek>   |
+    | inOnderzoek.datumIngangOnderzoek          | 20020701                    |
+
+    Voorbeelden:
+    | aanduiding in onderzoek | fields                                                 | type                          |
+    | 080000                  | verblijfplaats                                         | hele categorie verblijfplaats |
+    | 081200                  | verblijfplaats.type                                    | hele groep locatie            |
+    | 081210                  | verblijfplaats.functieAdres                            | locatiebeschrijving           |
+    | 088510                  | verblijfplaats                                         | datum ingang geldigheid       |
+    | 081020                  | verblijfplaats.type                                    | gemeentedeel                  |
+    | 080910                  | verblijfplaats                                         | gemeente van inschrijving     |
+    | 081410                  | verblijfplaats.functieAdres,verblijfplaats.inOnderzoek | land vanwaar ingeschreven     |
```

Comment:
Is dit niet exclusief de GBA gegevens die corresponderen met adressering (gemeentedeel, gemeente van inschrijving)? Hiervoor moet men dan toch met fields naar adressering vragen? Ik zie waarschijnlijk iets over het hoofd....